consensus/src/messages/mod.rs

```diff
@@ -248,3 +249,25 @@ pub struct ResponseTransactionReceiptsByAddress {
     #[beserial(len_type(u16, limit = 128))]
     pub receipts: Vec<(Blake2bHash, u32)>,
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RequestAccountsProof {
+    #[beserial(len_type(u16, limit = 128))]
+    /// Addresses for which the accounts trie proof is requested for
+    pub addresses: Vec<Address>,
+}
+
+impl RequestCommon for RequestAccountsProof {
+    type Kind = RequestMarker;
+    const TYPE_ID: u16 = 215;
+    type Response = ResponseAccountsProof;
+    const MAX_REQUESTS: u32 = MAX_REQUEST_TRANSACTIONS_PROOF;
```

Comment:
Are we sure we should reuse the TRANSACTIONS_PROOF limit for this request?

Reply:
Value is correct but name is not